logging/log_test.go

```diff
@@ -119,3 +119,20 @@ func TestSetJSONFormatter(t *testing.T) {
 	a.True(isJSON(bufNewLogger.String()))
 
 }
+
+func TestFatalExitHandler(t *testing.T) {
```

Comment:
please add a comment explaining why do we need this test. I.e. to ensure `RegisterExitHandler` is there and called by `Fatal` because of node graceful stop on fatal errors rely on this.